src/map/clif.c

```diff
@@ -14086,17 +14084,18 @@ static void clif_PartyBookingInsertNotify(struct map_session_data *sd, struct pa
 static void clif_PartyBookingUpdateNotify(struct map_session_data *sd, struct party_booking_ad_info *pb_ad)
 {
 #ifndef PARTY_RECRUIT
-	int i;
-	uint8 buf[6+PARTY_BOOKING_JOBS*2];
+	uint8 buf[6 + MAX_PARTY_BOOKING_JOBS * 2];
 
 	nullpo_retv(sd);
-	if(pb_ad == NULL) return;
+	if (pb_ad == NULL) 
+		return;
```

Comment:
```suggestion
	nullpo_retv(pb_ad);
```
And move nullpo checks to top.

Reply:
updated